frontend/src/pages/users/User.tsx

```diff
@@ -112,7 +112,17 @@ const UserComponent: FC<Props> = ({ user, refetch }) => {
   };
   const regenerateAPIKeyModal = showRegenerateAPIKey && (
     <Modal
-      message="Are you sure you want to regenerate the API key? This operation cannot be undone."
+      body={
```

Comment:
Any reason not to just use the "children" as the body?

Reply:
Just didn't think of it. 😅